src/offcanvas/offcanvas.js

```diff
@@ -73,8 +73,17 @@ angular.module("mm.foundation.offcanvas", [])
             require: '^offCanvasWrap',
             restrict: 'C',
             link: function ($scope, element, attrs, offCanvasWrap) {
-                element.on('click', function () {
-                    offCanvasWrap.hide();
+                element.find('li').on('click', function (e) {
```

Comment:
the solution has another drawback. When dynamically creating the menu, this solution does not work.